pkg/reconciler/route/table_test.go

```diff
@@ -2729,6 +2729,85 @@ func TestReconcile_EnableAutoTLS(t *testing.T) {
 		},
 		Key:                     "default/becomes-ready",
 		SkipNamespaceValidation: true,
+	}, {
+		// This test is a same with "public becomes cluster local" above, but confirm it does not create certs with autoTLS for cluster-local.
+		Name: "public becomes cluster local",
```

Comment:
Just noticed this, perhaps should use a different name so it doesn't conflict with the existing `public becomes cluster local` test. I know you have a comment here mentioning that, but would be good to differentiate during a test failure.

Reply:
Sure! Updated.